keyboards/keyten/kt60hs_t/info.json

```diff
@@ -0,0 +1,94 @@
+{
+    "manufacturer": "keyten",
+    "keyboard_name": "kt60HS-T",
+    "maintainer": "key10iq",
+    "processor": "STM32F401",
+    "bootloader": "stm32-dfu",
+    "usb": {
+        "vid": "0xEB69",
+        "pid": "0x6004",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["B15", "B14", "B12", "B13", "B0"],
+        "cols": ["A3", "A4", "A5", "A6", "A7", "B9", "B8", "B7", "B6", "B5", "B4", "B3", "A15", "A14"]
+    },
+    "community_layouts": ["60_tsangan_hhkb"],
+    "layouts": {
```

Comment:
Does this PCB support a WKL layout?

Reply:
yeah

Reply:
There is no support for the `60_ansi_tsangan` and `60_ansi_wkl` layouts though, because the PCB does not support 2.75u right Shift (only the 1.75u+1u split right Shift is supported).

The `60_ansi_wkl_split_bs_rshift` layout could be supported though, and `60_hhkb` too.

And it could make sense to add at least one layout with the 2u Backspace (it would need to be a custom one though — maybe `60_ansi_tsangan_split_rshift`, because there are no community 60% ANSI layouts which have split right Shift and 2u Backspace).

Reply:
suggestion added